code/modules/library/bookcase.dm

```diff
@@ -95,62 +95,80 @@
 	update_appearance()
 
 /obj/structure/bookcase/attackby(obj/item/I, mob/user, params)
```

Comment:
while you're reworking this entire slab of code, do you think it would be fruitful to change the argument to what it actually is on parent, `attacking_item`? would be well appreciated if you did